packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -1120,14 +1120,118 @@ describe("calcite-combobox", () => {
+      await item2.click();
+      await item3.click();
+
+      await element.click();
+      await element.press("Backspace");
+      const chips = await page.findAll("#myCombobox >>> calcite-chip");
+      expect(chips.length).toEqual(2);
+      expect((await element.getProperty("selectedItems")).length).toBe(2);
+    });
+
+    it("should not delete any items on Delete in single selection-display mode", async () => {
+      const page = await newE2EPage();
+      await page.setContent(html`
+        <calcite-combobox id="myCombobox" selection-display="single">
+          <calcite-combobox-item id="one" value="one" label="one"></calcite-combobox-item>
+          <calcite-combobox-item id="two" value="two" label="two"></calcite-combobox-item>
+          <calcite-combobox-item-group label="Last Item">
+            <calcite-combobox-item id="three" value="three" label="three"></calcite-combobox-item>
+          </calcite-combobox-item-group>
+        </calcite-combobox>
+      `);
+
+      const element = await page.find("#myCombobox");
+      await element.click();
+
+      const item1 = await page.find("calcite-combobox-item#one");
+      const item2 = await page.find("calcite-combobox-item#two");
+      const item3 = await page.find("calcite-combobox-item:last-child");
+      await item1.click();
+      await item2.click();
+      await item3.click();
+
+      await element.click();
+      await element.press("Backspace");
+      expect((await element.getProperty("selectedItems")).length).toBe(3);
+    });
+
+    it("should not delete any items on Delete in fit selection-display mode when there are overflowed chips", async () => {
+      const page = await newE2EPage();
+      await page.setContent(html`
+        <calcite-combobox id="myCombobox" selection-display="fit" style="width:350px">
+          <calcite-combobox-item id="one" value="one" text-label="one"></calcite-combobox-item>
+          <calcite-combobox-item id="two" value="two" text-label="two"></calcite-combobox-item>
+          <calcite-combobox-item-group text-label="Last Item">
+            <calcite-combobox-item id="three" value="three" text-label="three"></calcite-combobox-item>
+          </calcite-combobox-item-group>
+        </calcite-combobox>
+      `);
+
+      const element = await page.find("#myCombobox");
+      await element.click();
+
+      const item1 = await page.find("calcite-combobox-item#one");
+      const item2 = await page.find("calcite-combobox-item#two");
+      const item3 = await page.find("calcite-combobox-item:last-child");
+      await item1.click();
+      await item2.click();
+      await item3.click();
+
+      await element.click();
+      await element.press("Backspace");
+      expect((await element.getProperty("selectedItems")).length).toBe(3);
+    });
+
+    it("should delete last item on Delete in fit selection-display mode when there are no overflowed chips", async () => {
+      const page = await newE2EPage();
```

Comment:
It looks like there's a lot of common markup between the tests, so could you look at generalizing page setup logic in a `beforeEach` block to reduce duplication?